pkgs/development/python-modules/jxlpy/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  cython,
+  libjxl,
+}:
+buildPythonPackage rec {
+  pname = "jxlpy";
+  version = "0.9.5";
+
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-Kqdm8b3hgO0Q3zE68rSIM4Jd7upjG+SQywSseGwCFUI=";
+  };
+
+  build-system = [
+    setuptools
+    cython
+  ];
+
+  buildInputs = [ libjxl ];
+
+  pythonImportsCheck = [ "jxlpy" ];
+
+  meta = {
+    description = "Cython bindings and Pillow plugin for JPEG XL";
+    homepage = "https://github.com/olokelo/jxlpy";
```

```suggestion
    homepage = "https://github.com/olokelo/jxlpy";
    changelog = "https://github.com/olokelo/jxlpy/releases/tag/${version}";
```
Adding `meta.changelog` is helpful for future updates.